docker/Dockerfile

```diff
@@ -35,13 +32,12 @@ RUN wget -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-instal
 RUN pyenv install -v ${PYTHON_VERSION}
 RUN pyenv global ${PYTHON_VERSION}
 
-# Install CUDNN + TensorRT + dependencies
+# Install TensorRT + dependencies
 RUN wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/cuda-ubuntu2204.pin
 RUN mv cuda-ubuntu2204.pin /etc/apt/preferences.d/cuda-repository-pin-600
 RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/7fa2af80.pub
 RUN add-apt-repository "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/ /"
 RUN apt-get update
```

Comment:
This part can be removed I think. Reference: https://github.com/pytorch/TensorRT/blob/2a4c37bc5455b0fe3bb74d5ed7b57ef809c5198e/docker/Dockerfile#L33-L43

Reply:
You can leave it like this. I've removed this as a part of https://github.com/pytorch/TensorRT/pull/2793